x/superfluid/keeper/msg_server.go

```diff
@@ -2,13 +2,18 @@ package keeper
 
 import (
 	"context"
+	"errors"
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	lockuptypes "github.com/osmosis-labs/osmosis/v7/x/lockup/types"
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/types"
 )
 
+// TODO: fix upgrade height
+const v8UpgradeHeight = 721_000
```

Comment:
we should use and reference a single canonical const here. Probably the one defined with the fork code.

Reply:
should be solved via merging https://github.com/osmosis-labs/osmosis/pull/1483